### ISSUE TITLE
BUG: UserMembership query, doesn't return users part of groups of organisations

### ISSUE THREAD

**Issue by Comoque1** [alkem-io/server#1248]
**Describe the bug**
UserMembership query, doesn't return users part of groups of organisations.

**To Reproduce**
1.  Authenticate with global admin and go to [organisation group](https://dev.alkem.io/admin/organizations/6a940520-8e1f-478a-9f39-4de534c12fbb/groups/4ad11391-9e92-443e-bc35-52126c04f6ba/members)    
2. Add a User
3. Using the following query check the result:
```
query membershipUser($membershipData: MembershipUserInput!) {
  membershipUser(membershipData: $membershipData) {
    organisations {
      displayName 
      userGroups {
        displayName
      }
    }
  }
}
```
```
{
  "membershipData":{
    "userID": "comoque@gmail.com"
  }
}
```
4. The added user is not returned from the memberShip query
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/13135e6c-1182-4289-856c-b5179b4809dd)

**Expected behavior**
Membership query, must return user from organisation groups.

**Comment by techsmyth** [comment]
Is the user also a member of the organisation? If not then the group membership will not be shown as there is no org membership. 

Note: the UI + logic for the group should be updated to only allow adding users to groups from the parent org (or community)

**Comment by techsmyth** [comment]
to be addressed via #874

**Comment by Comoque1** [comment]
Tested OK on DEV env.
![image](https://user-images.githubusercontent.com/23620658/130101242-9778d9a6-f714-4560-8186-ef1c06910cc0.png)
